src/ToricVarieties/NormalToricVarieties/constructors.jl

```diff
@@ -16,6 +16,7 @@ export AffineNormalToricVariety
 function pm_object(v::AbstractNormalToricVariety)
     return v.polymakeNTV
 end
+export pm_object
```

Comment:
Why export? If it is not necessary I would like to hide this away.

Reply:
I stumbled over this a dozen times when developing code for the `ToricVarieties`.

There, we use `pm_object(v)` all the time. Yet, in an "ordinary" Julia session (where I am trying out my code - quicker than blindly running the tests) I have to write `Oscar.pm_object(v)`. This I forget all the time (or I add it in the "ordinary" Julia session, accidentally also add it in the implementation in Oscar and eventually have to chase these instances and delete "Oscar." there)...

This is an attempt to make the development more convenient for me. But maybe there is a better way?

Reply:
At the REPL you could do `import Oscar: pm_object` or `pm_object = Oscar.pm_object`.